zschema/tests.py

```diff
@@ -114,12 +114,32 @@ def test_invalid(self):
         "type": "INTEGER",
         "name": "ip",
         "mode": "NULLABLE"
-    }
+    },
 ]
 
 
 class CompileAndValidationTests(unittest.TestCase):
 
+    def assertBigQuerySchemaEqual(self, a, b):
+        """
+        Assert that the given BigQuery schemas are equivalent.
+
+        BigQuery schemas consist of lists whose order doesn't matter, dicts,
+        and privimites.
+        """
+        if type(a) is not type(b):
+            return False
```

Comment:
I'm not sure why this returns instead of asserting here.

Reply:
I'm also concerned that this is more strict than the `==` comparator in Python.

For example, ```>>> "a" == u"a"
True
>>> type("a") == type(u"a")
False```